src/routes/(console)/organization-[organization]/billing/paymentHistory.svelte

```diff
@@ -61,27 +66,40 @@
     $: if (offset !== null) {
         request();
     }
+
+    const columns = [
+        { id: 'dueDate' },
+        { id: 'status', width: { min: 200 } },
+        { id: 'amount', width: { min: 120 } },
+        { id: 'action', width: 40 }
+    ];
 </script>
 
 <CardGrid>
     <svelte:fragment slot="title">Payment history</svelte:fragment>
     Transaction history for this organization. Download invoices for more details about your payments.
     <svelte:fragment slot="aside">
-        {#if invoiceList.total > 0}
-            <Table.Root
-                let:root
-                columns={[
-                    { id: 'dueDate' },
-                    { id: 'status', width: { min: 200 } },
-                    { id: 'amount', width: { min: 120 } },
-                    { id: 'action', width: 40 }
-                ]}>
+        {#if invoiceList.total > 0 || isLoadingInvoices}
+            <Table.Root let:root {columns}>
                 <svelte:fragment slot="header" let:root>
                     <Table.Header.Cell column="dueDate" {root}>Due date</Table.Header.Cell>
                     <Table.Header.Cell column="status" {root}>Status</Table.Header.Cell>
                     <Table.Header.Cell column="amount" {root}>Amount due</Table.Header.Cell>
                     <Table.Header.Cell column="action" {root} />
                 </svelte:fragment>
+
+                {#if isLoadingInvoices}
+                    {#each Array.from({ length: 2 }) as _}
```

Comment:
i would just hardcode it, the Array.from approach requires type workarounds and values are always empty.

Reply:
feels a bit cleaner and why do we care about type, or the value? its X amount of rows loading skeletons.

Reply:
because svelte 5 actually wants us to have keys for each. also a `_` is not that clean :P

we would need to do:

```suggestion
                    {#each Array.from({ length: 2 }).keys() as index (index)}
```

to have it clean ;-)